### ISSUE TITLE
Update to Gutenberg 18

### ISSUE THREAD

**Issue by fjorgemota** [Automattic/isolated-block-editor#254]
As far as I know, we should probably just update the packages/dependencies.

**Comment by fullofcaffeine** [comment]
FWIW, the latest version works well when Gutenberg 18-assets are enqueued in the WordPress env when the plugin is loaded. I think that's due to most core package's assets being extracted out using the `DependencyExtractionPlugin`, so the iso editor ends up using what's available at runtime, which are the versions published/packaged with GB v18.0.0.

If that's the case, that means we might delay the update to the iso editor, and updating the GB plugin should be enough for most cases, unless the interface changes drastically and the consuming code here breaks because of that (though that shouldn't happen unless we're using unstable APIs, I think).

That being said, I believe we should still aim to align the packages consumed here with the most recent versions available - this could probably be partially automated with renovate?

**Comment by fjorgemota** [comment]
Yeah, I agree. I created this issue also as a way to track any possible incompatibilities, but, if they do not exist, then just updating the packages should be enough.

> this could probably be partially automated with renovate?

I'm not sure. I usually align the versions with the versions present at the moment of the release. I'm not sure what's the best approach here.
